### PR TITLE
Turn on schema.org when serializing

### PR DESCRIPTION
**Pull request by ylt**

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -81,38 +81,40 @@
...
                 "type": "zip",
-                "url": "https://api.github.com/repos/doctrine/instantiator/zipball/8e884e78f9f0eb1329e445619e04456e64d8051d",
-                "reference": "8e884e78f9f0eb1329e445619e04456e64d8051d",
+                "url": "https://api.github.com/repos/doctrine/instantiator/zipball/ae466f726242e637cebdd526a7d991b9433bacf1",
+                "reference": "ae466f726242e637cebdd526a7d991b9433bacf1",
                 "shasum": ""
             },
             "require": {
-                "php": ">=5.3,<8.0-DEV"
+                "php": "^7.1"
```
Comment (thtg88) [inline composer.lock]: Have the PHP version requirements changed? 🤔

Comment (ylt) [reply composer.lock]: Not sure why that happened...

Comment (thtg88) [reply composer.lock]: Might be a random `composer update` / `composer install` from a PHP 7.* 🤔

Comment (ylt) [reply composer.lock]: Yep, looks like it. It's just the dev requirements, the main requirements still allow both PHP versions.

Comment (thtg88) [reply composer.lock]: That's perfect then - all good for me

**Pull request merged**